### DISCUSSION TITLE
How often is the release?

### DISCUSSION THREAD

**Discussion by johnmckenna** [nx-go/nx-go#135] [Q&A] [answered]
How often does nx-go release?

If it's manual, do you need any help automating this?

Do you want any additional maintainers?

**Comment by utarwyn** [comment, marked as answer]
Hello @johnmckenna!

There is no release cycle at the moment for nx-go because I maintain the project on my free time. I was also quite busy the last couple of weeks because of a job change. If needed I can review pull requests, so additional maintainers are really appreciated to address opened issues 🙏

I'll be able to work more actively on the project by the end of september.
Does this answer your question?

**Reply by johnmckenna** [reply]
Thanks @utarwyn Best of luck in your new role!

Just relying on this for the go test timeout for Terratest. So would be great to get a release for that.

Happy to help if I can.
